examples/2016-10-31/iot_backend/template.yaml

```diff
@@ -0,0 +1,27 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: IoT -> Lambda -> DynamoDB
+Resources:
+  IoTFunction: 
+    Type: AWS::Serverless::Function
+    Properties: 
+      Handler: index.handler
+      Runtime: nodejs4.3
+      CodeUri: ./iot_backend.zip
```

Comment:
To be consistent with other examples, set CodeUri to s3://<bucket>/iot_backend.zip